.github/workflows/ci.yml

```
@@ -505,14 +504,18 @@ jobs:
         engine: ["python", "ray", "dask"]
     env:
       MODIN_ENGINE: ${{matrix.engine}}
-      MODIN_MEMORY: 1000000000
       # Only test reading from SQL server and postgres on ubuntu for now.
       # Eventually, we should test on Windows, too, but we will have to set up
       # the servers differently.
       MODIN_TEST_READ_FROM_SQL_SERVER: true
       MODIN_TEST_READ_FROM_POSTGRES: true
     name: test-ubuntu (engine ${{matrix.engine}}, python ${{matrix.python-version}})
     steps:
+      - name: Limit ray memory
+        run: |
+          if [ "${{matrix.engine}}" = "ray" ]; then
+            echo "MODIN_MEMORY=1000000000" >> $GITHUB_ENV
+          fi
```

```suggestion
        run: echo "MODIN_MEMORY=1000000000" >> $GITHUB_ENV
        if: matrix.engine == 'ray'
```
this looks cleaner IMHO

here and in other places